machinelearning/onnx.md

```diff
@@ -0,0 +1,102 @@
+# ML inference on Graviton CPUs with Open Neural Network Exchange(ONNX)
+
+**Introduction**
+
+ONNX is an open-source machine learning framework that provides interoperability between different frameworks. ONNX Runtime is the runtime engine used for model inference and training with ONNX . This document covers how to use ONNX based machine learning inference on Graviton CPUs, what runtime configurations are important and how to run benchmarking scripts. The document also covers instructions for source builds to enable experimenting with downstream features.
+
+# How to use ONNX on Graviton CPUs
+
+Python wheels is the recommended option to use ONNX on Graviton, the default backend is optimized for Graviton CPU.
+
+```
+# Install Python
+sudo apt-get update
+sudo apt-get install -y python3 python3-pip
+
+# Upgrade pip3 to the latest version
+python3 -m pip install --upgrade pip
+
+# Install ONNX and ONNX Runtime
+python3 -m pip install onnx
+python3 -m pip install onnxruntime
+```
```

Comment:
Is Ubuntu the recommended distribution?  Or should you provide instructions for AL2023 instead?

Reply:
thanks for spotting it. Both Ubuntu and AL2023 are recommended, I will remove the package installer specifics, anyway python installation details are covered separately in Graviton user guide.